man/systemd.exec.xml

```diff
@@ -3694,6 +3694,20 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
           system.</para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>$TRIGGER_UNIT</varname></term>
+          <term><varname>$TRIGGER_PATH</varname></term>
+
+          <listitem><para>If the unit was activated dynamically (e.g.: a corresponding path unit), the
+          unit that triggered it and other type-dependent information will be passed via these variable. Note that
+          this is provided in a best-effort way. For example, there might be multiple triggers coalesced if happening
+          one after the other, and only one will be reported, and there is no guarantee on which one it will be.
+          Because of this, if there are multiple triggers configured for the same unit, this variable will be primarily
```

Comment:
"if there are multiple triggers"? There almost certainly are, after all users can always start services manually via "systemctl start". i'd thus be more vague here. maybe:

"Becuase of this, in most cases this variable will be primarily information, …"